packages/integration-sdk-cli/src/__tests__/cli-run.test.ts

```diff
@@ -61,6 +61,25 @@ test('enables graph object schema validation', async () => {
   expect(process.env.ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION).toBeDefined();
 });
 
+test('disables graph object schema validation', async () => {
+  const job = generateSynchronizationJob();
+
+  setupSynchronizerApi({ polly, job, baseUrl: 'https://api.us.jupiterone.io' });
+
+  expect(process.env.ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION).toBeUndefined();
+
+  await createCli().parseAsync([
+    'node',
+    'j1-integration',
+    'run',
+    '--integrationInstanceId',
+    'test',
+    '--disable-schema-validation',
+  ]);
+
+  expect(process.env.ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION).toBeUndefined();
```

Comment:
I'd really like if the assertions were against real runtime behavior instead of the value of an environment variable value. Tbh, I don't like that we mutate the value of `ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION` in the first place. Ideally, we would remove `ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION` (breaking) completely and only support the CLI arg.

Reply:
What would your preference be for this effort?  Go ahead and close this task out and capture a new task to remove `ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION` completely or go ahead and make those changes in this PR?

Reply:
My preference is that we'd make the suggested changes separately. I do think that we should consider improving the test added in this PR to actually assert that schema validation is not being considered _instead_ of just checking the value of an environment variable. 

Take a look at how this test asserts that the step `status` is `SUCCESS` because schema validation is disabled: https://github.com/JupiterOne/sdk/blob/1280faf454cefa6aff1aabd90b4890c1920b88c9/packages/integration-sdk-cli/src/__tests__/cli.test.ts#L215

Reply:
I left the existing `ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION` tests in place, but added two additional tests that actually test for SUCCESS/FAILURE as outlined in the linked tests.  Thanks!